media/starboard/starboard_renderer.cc

```diff
@@ -644,6 +645,8 @@ void StarboardRenderer::CreatePlayerBridge() {
     player_bridge_->SetVolume(volume_);
 
     state_ = STATE_FLUSHED;
+    base::MemoryPressureListener::NotifyMemoryPressure(
```

Comment:
micro-nit: Could we leave a comment here to clarify the
meaning? E.g.
```
  // Send a one-time critical memory pressure signal to ask
  // other components to release memory.
```
or some such?